GameCube/source/game_patch/02_enemybgm.cpp

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file 02_enemybgm.h
+ * @author jdflyer
+ * @brief Patches the game to allow for optional enemy BGM
+ *
+ * @bug No known bugs
+ */
+#include "game_patch/02_enemybgm.h"
+
+#include "main.h"
+#include "patch.h"
+#include "tp/Z2SeqMgr.h"
+
+namespace mod::game_patch::bgm::enemybgm
+{
+    void ( *startBattleBgmTrampoline )( libtp::tp::z2audiolib::z2seqmgr::Z2SeqMgr* seqMgr, bool param_1 ) = nullptr;
```

Comment:
hould follow the naming scheme "return_[originalFunctionName]" for consistency (check main.h/main.cpp)